src/lib/InputProcessor.ts

```diff
@@ -80,3 +112,38 @@ function outputFormatFromInputs(inputs: Inputs): OutputFormat {
 		return OutputFormat.TABLE;
 	}
 }
+
+function trimMethodSpecifier(targetPath: string): string {
+	return targetPath.split('#')[0];
```

Comment:
Question: Should this maybe be something involving `lastIndexOf()`, in case we eventually want some kind of workaround to let users have \#'s in their file names?

Reply:
Good point. Unfortunately, this is exactly how we implement things in the DefaultRuleManager... so I wanted to be consistent. If we want to support this, we can do a follow up work item to fix this in both places.

Reply:
OK i'll fix it in both places... seems simple enough once I got in there.